pages/admin/users/index.tsx

```diff
@@ -13,8 +241,80 @@ const AdminUsers: NextPage = () => {
+                </div>
+                <div className="relative w-96">
+                  <div className="mt-1 relative fill-stone-400">
+                    <DebouncedInput
+                      type="text"
+                      value={
+                        (table
+                          .getColumn('firstName')
+                          .getFilterValue() as string) ?? ''
+                      }
+                      onChange={(value) =>
+                        table
+                          .getColumn('firstName')
+                          .setFilterValue(String(value))
+                      }
+                      placeholder="Søk etter bruker"
+                      className="w-full py-2 px-4 pl-10 border-2 border-stone-300 outline-none text-sm text-left leading-6 bg-transparent rounded-xl transition duration-150 ease-in-out"
+                    />
+                    <Search
+                      className="w-4 h-4 absolute top-[13px] left-4"
+                      color="inherit"
+                    />
+                  </div>
+                </div>
+              </div>
+
+              <div className="bg-white rounded-xl w-full h-full p-6">
+                <div className="flex justify-between items-center pb-6">
+                  <div>
+                    <p
+                      className={`text-sm text-neutral-500 transition-all duration-500 ${
+                        Object.keys(rowSelection).length > 0
+                          ? 'opacity-100'
+                          : 'opacity-0'
+                      }`}
+                    >
+                      {Object.keys(rowSelection).length} av{' '}
+                      {table.getPreFilteredRowModel().rows.length} valgt
+                    </p>
+                  </div>
+                  <Link href="/admin/users/new">
+                    <a>
+                      <Button
+                        text="Legg til bruker"
+                        className="text-xs py-3 px-8"
+                      />
+                    </a>
+                  </Link>
+                </div>
+                <div className="grid [grid-template-rows:minmax(409px,1fr)_50px]">
+                  <div className="rounded-lg border border-neutral-300 overflow-hidden">
+                    <AdminTable table={table} />
+                  </div>
+                  <div className="flex justify-between items-end gap-2">
+                    <p className="flex items-center gap-1 text-sm">
+                      Viser
+                      <strong>
+                        {1 + pageIndex * pageSize} -{' '}
+                        {pageIndex + 1 == pageCount
+                          ? users?.length
+                          : (pageIndex + 1) * pageSize}{' '}
+                        av {users?.length}
+                      </strong>
+                      brukere
+                    </p>
+                    <AdminTablePagination table={table} />
```

Comment:
Vi bør finne ut hvordan vi kun henter 9 brukere fra db om gangen og ikke alle med en gang. Prisma har sånn take og skip for pagination.

Reply:
Yep, det skal være mulig ja. Se https://github.com/vsaIT/vsait.org/blob/main/pages/events/index.tsx for eksempel på db querying med pagination. Her bruker jeg useQuery though :smirk: